test/pytorch_test_base.py

```diff
@@ -275,6 +275,11 @@
         'test_Dropout2d_xla',  # Started to pass
     },
 
+    # test/nn/test_pooling.py
+    'TestPoolingNNDeviceTypeXLA': {
+        'test_pool_invalid_size',  # expecting a different runtime error
```

Comment:
can you remove the same entry in `test_nn` above?

Reply:
Ah. Right. (And sorry I forgot yesterday was a holiday 😅)